### PR TITLE
Add logic for putting mpi_id only for children on declaration_request create

### PR DESCRIPTION
**Pull request by progsmile**
#3092

### CODE REVIEW

File: apps/core/lib/core/declaration_requests/api/v1/creator.ex [hunk 1]
```diff
@@ -691,12 +694,27 @@ defmodule Core.DeclarationRequests.API.V1.Creator do
...
+    if child?(person_birth_date) do
+      changeset
+    else
+      put_change(changeset, :mpi_id, person_id)
+    end
+  end
+
+  defp child?(birth_date) do
+    case Date.from_iso8601(birth_date) do
+      {:ok, birth_date} -> Timex.diff(Timex.now(), birth_date, :years) < 14
```
Comment (imax-iva) [inline apps/core/lib/core/declaration_requests/api/v1/creator.ex]: does it make sense to put a variable in the configuration?

File: apps/core/lib/core/declaration_requests/api/v1/creator.ex [hunk 2]
```diff
@@ -691,12 +696,27 @@ defmodule Core.DeclarationRequests.API.V1.Creator do
...
+    if passport_age?(person_birth_date) do
+      changeset
+    else
+      put_change(changeset, :mpi_id, person_id)
+    end
+  end
+
+  defp passport_age?(birth_date) do
+    case Date.from_iso8601(birth_date) do
+      {:ok, birth_date} -> Timex.diff(Timex.now(), birth_date, :years) < @years_to_have_passport
```
Comment (imax-iva) [inline apps/core/lib/core/declaration_requests/api/v1/creator.ex]: it seems there is a need to reverse condition:
Timex.diff(Timex.now(), birth_date, :years) > @years_to_have_passport
and change logic in put_mpi_id function

File: apps/core/lib/core/declaration_requests/api/v1/creator.ex [hunk 3]
```diff
@@ -42,6 +42,8 @@ defmodule Core.DeclarationRequests.API.V1.Creator do
   @status_new DeclarationRequest.status(:new)
   @status_approved DeclarationRequest.status(:approved)
 
+  @years_to_have_passport 14
```
Comment (imax-iva) [inline apps/core/lib/core/declaration_requests/api/v1/creator.ex]: there is a sense to use config variable instead of hardcoded value

Comment (progsmile) [reply apps/core/lib/core/declaration_requests/api/v1/creator.ex]: across code

**Pull request merged**